powershell/SecretSanta.ps1

```diff
@@ -0,0 +1,43 @@
+﻿# SecretSanta.ps1
+# Script that accepts list of names and then determines a secret
+# santa match for each person
+
+Param(
+    [Parameter(Mandatory=$true)][string[]]$names
+)
+
+# Create function that returns $true if for every value in $listA,
+# the corresponding value at the same position in $listB is not the 
+# same. Returns $false otherwise
+Function Compare-Lists ( [string[]]$listA, [string[]]$listB )
+{
+    $i = 0
+    ForEach ($item in $listA) {
+        If ($listB[$i] -eq $item) {
+            $false
+            Return
+        }
+        $i++
```

Comment:
Is there a better way to do the loop? It is a bit odd from a `Java` viewpoint to use the forEach loop and then increment each time to index into the second array.

I know in python there is a useful method called `enumerate` that does a `for each` loop, but also returns the index. 

I noticed on line 40 the similar `forEach` loop with variable is used. I wonder if there is a more clean way to do both of these loops.